src/app/dashboard-widgets/create-work-item-widget/create-work-item-overlay/create-work-item-overlay.component.ts

```diff
@@ -43,7 +45,7 @@ export class CreateWorkItemOverlayComponent implements OnInit, AfterViewInit {
   }
 
   onClose() {
-    this.router.navigateByUrl(removeAction(this.router.url));
+    this.location.back();
```

Comment:
When choosing to create a work item, I encounter the error below. I don't see the overlay, but an empty page that says "Choose a new work item type".

ERROR Error: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'false'. Current value: 'true'.